CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+
+#### Before Submitting An Enhancement Suggestion
+
+* You might discover that the enhancement is already available. Most importantly, check if your clone is even with GeoJson To SVG CLI's master branch.
+* Check if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on GeoJson To SVG CLI, and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of GeoJson To SVG CLI which the suggestion is related to. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
+* **Explain why this enhancement would be useful** to most users.
+* **Specify which version of GeoJson To SVG CLI you're using.** You can get the exact version by running `hugo version` in your terminal.
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to GeoJson To SVG CLI? You can start by looking through these `beginner` and `help-wanted` issues:
+
+* [Beginner issues][beginner] - issues which should only require a few lines of code, and a test or two.
+* [Help wanted issues][help-wanted] - issues which should be a bit more involved than `beginner` issues.
+
+Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have.
+
+### Local development
+
+For working on GeoJson To SVG CLI, you can simply clone [this repository](https://github.com/derhuerst/geojson-to-svg-cli), and make changes to its source code. To ensure if your changes work, test them against the [`example site`](/exampleSite).
+
+### Pull Requests
+
+* Fill in [the required template](PULL_REQUEST_TEMPLATE.md)
```

Comment:
> PULL_REQUEST_TEMPLATE.md

Is this an actual file in this repo? Or is it some GitHub magic? Because I don't see the file here.